content/github/setting-up-and-managing-your-enterprise/adding-organizations-to-your-enterprise-account.md

```diff
@@ -11,7 +11,7 @@ versions:
 
 Enterprise owners can create new organizations within an enterprise account's settings.
 
-To add an organization to your enterprise account, you must create the organization from within the enterprise account settings. You cannot transfer existing organizations into an enterprise account.
+To add an organization to your enterprise account, you must create the organization from within the enterprise account settings. You cannot transfer existing organizations into an enterprise account on your end. If you need assistance transferring an existing organization please contact your GitHub Sales account representative.
```

Comment:
For clarity, I think we should avoid using "on your end". I think we could just cut the existing second sentence and just mention contacting GitHub. How about:

```suggestion
To add an organization to your enterprise account, you must create the organization from within the enterprise account settings. If you want to transfer an existing organization to your enterprise account, contact your {% data variables.product.prodname_dotcom %} sales account representative.
```

I wonder whether everyone reading this will know who their sales account rep is. Would it be better to tell them to contact Support, as we do for many other things? 

i.e. 
> contact [{% data variables.product.prodname_dotcom %} Support](https://support.github.com/contact).